pages.id/linux/apt-file.md

```diff
@@ -0,0 +1,20 @@
+# apt-file
+
+> Mencari file-file di dalam paket `apt`, termasuk yang belum diinstal.
```

Comment:
```suggestion
> Cari kumpulan berkas di dalam paket `apt`, termasuk yang belum dipasang.
```

Can you please change this (and subsequent) sentences to use imperative mood (see [this](https://share.reinhart1010.id/tldrguidelinesidspecific) and [this](https://share.reinhart1010.id/tldridimperativemood))?

Additionally, as a maintainer I currently intend to deprecate `file` in favor of `berkas` for Indonesian translations.